modules/administration-guide/pages/repairing-the-devworkspace-operator-on-openshift.adoc

```diff
@@ -0,0 +1,80 @@
+:_content-type: PROCEDURE
+:navtitle: Repairing the {devworkspace} Operator on OpenShift
+:description: Repairing the {devworkspace} Operator on OpenShift
+:keywords: devworkspace, operator, repair
+:page-aliases:
+
+[id="repairing-the-devworkspace-operator-on-openshift_{context}"]
+= Repairing the {devworkspace} Operator on OpenShift
+
+Under certain conditions, such as https://docs.openshift.com/container-platform/4.10/operators/understanding/olm/olm-understanding-olm.html[OLM] restart or cluster upgrade, the {prod2} Operator for {prod-short} might automatically install the {devworkspace} Operator even when it is already present on the cluster. In that case, you can repair the {devworkspace} Operator on OpenShift as follows:
+
+.Prerequisites
+
+* An active `{orch-cli}` session as a cluster administrator to the destination OpenShift cluster. See {orch-cli-link}.
+
+* On the *Installed Operators* page of the {platforms-name} web console, you see multiple entries for the {devworkspace} Operator or one entry that is stuck in a loop of *Installing* and *InstallSucceeded*.
```

Comment:
According to another comment, the two states are `'Replacing' and 'Pending'`, not `*Installing* and *InstallSucceeded*`.

Reply:
Should I include both pairs?

Reply:
no, Nick is correct it should be 'Replacing' and 'Pending' as I have mentioned in https://github.com/eclipse-che/che-docs/pull/2419#discussion_r946897678

Reply:
Fixing it to 'Replacing' and 'Pending'.

Reply:
@nickboldt, please note that I'm also updating this in the KI text in https://issues.redhat.com/browse/CRW-3258